discover/portal/articles/07-setting-up-liferay/01-system-settings/02-configuration-files.markdown

```diff
@@ -0,0 +1,151 @@
+configurations for other
+[scopes](/discover/portal/-/knowledge_base/7-0/configuring-liferay#configuration-scope)
+in @product@. 
+
+The System Settings application has an *Export* option. Once a configuration
+entry is modified, the export option becomes available, letting you download a
+`.config` file with all of the settings in a `key=value` format. Export is
+available for either a single configuration entry or the entire set of modified
+configurations in @product@. 
+
+If you don't have access to System Settings directly, or if you need portable
+configurations that can be deployed to multiple Liferay systems quickly and
+easily, without fumbling around in a UI for each one, configure @product@ with
+an OSGi configuration file. 
+
+There are several things to understand about configuration files: 
+
+- They're part of the [Apache Felix Configuration
+    Admin](http://felix.apache.org/documentation/subprojects/apache-felix-config-admin.html)
+    framework. Find out more about their syntax
+    [here](https://sling.apache.org/documentation/bundles/configuration-installer-factory.html#configuration-files-config). 
+- The `.config` file format supports a richer encoding format than the `.cfg`
+    file type, which only allows Strings. 
+- If the default values from System Settings are desirable, no configuration
+    file is necessary. 
+- System Settings exports all configurations keys and values for an entry, even if only one
+    of the values has been modified. 
+- If you place a configuration file in your @product@ installation's
+    `osgi/configs` folder, any changes you make to the entry in System Settings
+    will be automatically propagated to the configuration file. 
+
++$$$
+
+**Note:** Because the `.config` file supports types, under certain conditions
+special type marker characters are inserted into the files. For example, a
+configuration with a boolean type might have a `B` inserted just before the value
+to mark the type. 
+
+These type markers are neither necessary nor problematic. You can safely ignore
+them. @product@ doesn't rely on the configuration marker to determine what type
+the configuration is, because the backend code creating the setting already
+knows what the type must be. 
+
+In short, a `.config` file with 
+
+    addDefaultStructures=B"true"
+
+functions no differently than 
+
+    addDefaultStructures="true"
+
+in @product@. 
+
+$$$
+
+## Creating and Deploying a Configuration File [](id=creating-and-deploying-a-configuration-file)
+
+Configuration files consist of two things: 
+
+1.  A file name. 
+
+2.  One or more keys and values, with the value enclosed in double quotes. 
+
+        configurationName="value" 
+
+The name of the file uses the `id` of the configuration object specified in the
+backend Java code that creates the configuration (the class name will be
+`*Configuration.java` for readers interested in exploring the code). For
+example, 
```

Comment:
In the example, Is the `id` the part `com.liferay.journal.configuration.JournalService` of the name?